cmd/crc/cmd/config/config.go

```diff
@@ -49,6 +50,9 @@ func RegisterSettings(cfg *config.Config) {
 
 	// Telemeter Configuration
 	cfg.AddSetting(ConsentTelemetry, "", config.ValidateYesNo, config.SuccessfullyApplied)
+
+	// Change startup behaviour
+	cfg.AddSetting(StartOpenShift, true, config.ValidateYesNo, config.SuccessfullyApplied)
```

Comment:
Fwiw, it's not really intuitive to disable seemingly unrelated stuff ('openshift') in order to enable 'podman mode', maybe this should be `cluster-mode: podman|openshift` or something similar.

Reply:
Podman is not a mode; it is always existent.

Reply:
Would it be possible to compile crc under a new name like podman-machine and change slightly the behavior based on this ?
If I am using only podman, I use podman-machine but I can "upgrade" instantly by using "crc".

Reply:
On linux/mac, no need for a new binary, this can be a symlink + a check of os.Args[0]. Maybe this can also work on windows as well.

Reply:
> Podman is not a mode; it is always existent

Could be `podman-only` and `podman+openshift`. But ok, this `disable-openshift` toggle is not required for podman, it's just an optimization.

Reply:
This should be reviewed in isolation of podman
The start and stop of openshift is something that enables this, but it's not the only driver. 

Starting the vm without anything besides DHCP already is a podman vm with our image

Reply:
The idea could be to reintroduce `crc openshift [start/stop/status/...]`

Reply:
> Complexity of a new executable for what benefit? 

Why podman would be a second-class citizen and not openshift ? 
On Linux, users will use crc for openshift. 
On macOS, there is more value and more users to gain with podman.

For me the UX feels better with 2 commands. 

crc is a way to have a maintained docker-machine for podman nothing else.. With 2 separated binaries/or a clean way to differentiate both, I would bake a minimal bundle and create a Podman Desktop.pkg :)

Reply:
you would want to run two VMs too?

Reply:
In case of 2 differents apps, yes. Maybe for OKD also, we could be able to launch side by side 2 VMs.

Reply:
For me CRC is the 'brand' for single-user/evaluator OpenShift, so even if we agree on a parameter name to have any sort of 'crc --no-openshift' is entirely confusing. The same way that 'java --no-java' or 'chectl --no-che' would be.

It seems like the use of this would be fairly different too, so even if the implementation is shared or very similar I'd prefer to see a separate cli.

Reply:
The cost of 2 CLIs is really high.
Maybe we can start with this simple flag and see if people using it with the telemetry. If there is enough usage, we can turn that into a proper CLI ?

Reply:
Right,  it should not be underestimated what the cost involved is. It would be like a totally new lifecycle management tool. Even if shared in this codebase, the technical debt for that is very high.

These changes to have the cluster be restarted bring benefit by itself.

Note: this parameter we also used on minishift for a docker runtime only option.

Reply:
The question can still be; should this be part of CRC?